tests/operators/models/test_mono_exponential_decay.py

```diff
@@ -56,3 +57,11 @@ def test_mono_exponential_decay_shape(parameter_shape, contrast_dim_shape, signa
     m0, decay_constant = create_parameter_tensor_tuples(parameter_shape, number_of_tensors=2)
     (signal,) = model_op.forward(m0, decay_constant)
     assert signal.shape == signal_shape
+
+
+@pytest.mark.filterwarnings('ignore:Anomaly Detection has been enabled')
+def test_autodiff_exponential_decay():
+    """Test autodiff works for mono-exponential decay model."""
+    model = MonoExponentialDecay(decay_time=20)
+    m0, decay_constant = create_parameter_tensor_tuples(parameter_shape=(2, 5, 10, 10, 10), number_of_tensors=2)
+    autodiff_of_operator_test(model, m0, decay_constant)
```

Comment:
what is this error??

test_autodiff_exponential_decay

DeprecationWarning: ast.Str is deprecated and will be removed in Python 3.14; use ast.Constant instead

Reply:
> DeprecationWarning: ast.Str is deprecated and will be removed in Python 3.14; use ast.Constant instead

where did you get this error?

Reply:
It was auto annotated by the pytest action at the time I did the review (if there is no force push, you should be able to see it). Maybe a pytest update resolved the issue?